mkdocs.yml

```diff
@@ -0,0 +1,31 @@
+site_name: RESTier Documentation
+site_description: How to use the RESTier framework for .NET.
+theme: readthedocs
+pages:
+- Home:
+    - 'Introduction': 'index.md'
+    - 'Getting Started': 'getting-started.md'
```

Comment:
I was thinking number is better to reference like tell people which section, record note with section and so on.

If practice already shows not too much benefit, I am OK now.